### ISSUE TITLE
SourceForge site for FreeGLUT is getting blocked - consider migrating?

### ISSUE THREAD

**Issue by kozross** [FreeGLUTProject/freeglut#37]
Given that at least one adblocker (UBlock) that's in common use blocks SourceForge basically all the time, I'd be nice if you guys could migrate off it.

**Comment by rcmaniac25** [comment]
TL:DR-It's possible, but one case does not make it a necessity. But you should take it up on the FreeGLUT developer mailing list as there are some (I think) that use SVN instead of Git, but if they agree then it could happen.

Sourceforge certainly has issues, given the exodus of projects from it, SF has opted to flex it's EULA and modify certain installer downloads (often those that have abandoned SF). Given that the current FG downloads are not executables, it's (presumably) safe to assume that they are not modified automatically as the executables are.

Based on https://github.com/gorhill/uBlock/issues/583, the addon blocks it based on the fact that some downloads can be risky (as mentioned above). Not only does the uBlock author state that the addon will now point out that it could be risky to download from that site, it points out how to unblock it if desired.

Taking human nature into account, FreeGLUT is only useful for developers and (hopefully) is included with whatever executable requires it. If a developer encounters a blocked page, and they have no knowledge of Free/GLUT, they may unfortunately move on. Otherwise, there is a good chance they will assume it's a false positive and follow the link uBlock provides or know how to unblock it already. But there is also a high chance that many modern developers start a search from Google which does list the GitHub page (abit, lower then it would be useful). If they do the search direct from GitHub, they will get to this page.

Now, all of that is nice but it's also a cop-out and doesn't help the situation in regards to uBlock. It's unfortunate to be branded as risky for something that is not the project's fault and the choice of Sourceforge is a historic one. Between the long mailing lists of which various decisions about how the project works, some of which are referenced within the affected code blocks itself, it may not be desired to move. The docs can be moved (to an extent), and tickets might be able to be transferred, but one tough one is that there may be some who don't use Git and don't want to. Ignoring the opinion's of the developer(s), we need a fair amount of more help on FG and loosing help because a SVN branch went away due to one of many blocking tools grouping dev. projects into "risky" is not something the project would probably want to risk.

If you really want this and/or can probably provide additional cases/blockers where SF being blocked, bring it up with the FreeGLUT developer mailing list where the active contributors can be found, and if they agree to switch, then there is a good chance we'll switch. Otherwise, no harm in trying.

*Sorry for the long writeup, just providing a lot of context*

**Comment by nigels-com** [comment]
As the GLEW maintainer, I'm progressively shutting down things on SourceForge.  Just sayin'.

- Nigel

**Comment by rcmaniac25** [comment]
So I take that as there's a good chance we'll be switching GLUT off SourceForge at some point?

**Comment by dcnieho** [comment]
brought it up on the mailing list. Thanks Vinnie!

**Comment by jtsiomb** [comment]
I don't like sourceforge at all... but this is certainly NOT the reason to migrate.
